### PR TITLE
Improve chirp deletion

### PR DESCRIPTION
**Pull request by Xelowak**
I redesigned the delete button and added a modal to ask for user's confirmation before deleting.

New design
Capture d’écran 2022-01-29 à 17 09 04

Confirmation modal
Capture d’écran 2022-01-29 à 17 09 14
Clicking on yes deletes the chirp, and clicking on no dismisses the modal.

### CODE REVIEW

File: fe1-web/res/strings.ts [hunk 1]
```diff
@@ -68,6 +68,8 @@ const STRINGS = {
   follow_button: 'Follow',
   profile_button: 'Profile',
   social_media_your_profile_unavailable: 'You do not have a social media profile yet, be sure to have participated in a roll call.',
+  modal_chirp_deletion_title: 'Chirp deletion',
+  modal_chirp_deletion_description: 'Are you sure you want to delete this chirp ?',
```
Comment (jiabaow) [inline fe1-web/res/strings.ts:72]:
```suggestion
  modal_chirp_deletion_description: 'Are you sure you want to delete this chirp?',
```

**Review by jiabaow — APPROVED** [review]
LGTM
Thanks for the improvement!

**Review by pierluca — DISMISSED** [review]
Approving blindly

**Review by pierluca — APPROVED** [review]
Approving blindly (again 😄)

**Pull request merged**